src/features/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+import categoriesReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    categories: categoriesReducer,
+  },
+});
+
+export default store;
```

Comment:
- [ ] Good job on your work 👏 . However, kindly follow the `ducks pattern` [Link](https://github.com/erikras/ducks-modular-redux) for your Redux files and all file folders As instructed in the project requirements 👍 

### Expected:
![image](https://github.com/ivangonzalez224/Bookstore/assets/50990925/2884afaf-55c6-4cd6-aa1b-cd455538b4e0)

Reply:
Hi @Shedrack-Sunday ,
Thank you for the review, I don't quite understand the  ducks pattern very well, in this case do you mean that I just have to change the name of the folder called 'features' to 'redux'? or is there anything else to change ?